src/app/components/Curation/index.tsx

```diff
@@ -143,31 +141,31 @@ export default ({
     case SIMPLE_CURATION_GRID:
     case HIERARCHICAL_CURATION_GRID:
     default:
-      return curationLength > 1 &&
-        summaries.length > 0 &&
-        (title || isFirstCuration) ? (
-        <section aria-labelledby={id} role="region">
-          {isFirstCuration ? (
-            <VisuallyHiddenText id={id} as="h2">
-              {curationSubheading}
-            </VisuallyHiddenText>
-          ) : (
-            <Subheading id={id} link={link}>
-              {curationSubheading}
-            </Subheading>
-          )}
+      if (summaries.length > 0) {
+        return curationLength > 1 && (title || isFirstCuration) ? (
+          <section aria-labelledby={id} role="region">
+            {isFirstCuration ? (
+              <VisuallyHiddenText id={id} as="h2">
+                {curationSubheading}
+              </VisuallyHiddenText>
+            ) : (
+              <Subheading id={id} link={link}>
+                {curationSubheading}
+              </Subheading>
+            )}
+            <GridComponent
+              summaries={summaries}
+              headingLevel={3} // multiple curations, don't we want the promos to all be h3?
+            />
+          </section>
+        ) : (
           <GridComponent
             summaries={summaries}
-            headingLevel={isFirstCuration ? 3 : headingLevel}
+            headingLevel={2} // if there is only one curation, don't we always want promos to be h2?
```

Comment:
Let's update this comment (but yes, if there is only one curation, then all promo headings should be h2 😄 )
```suggestion
            headingLevel={2} // if there is only one curation, all promos should be h2
```